build.gradle

```diff
@@ -13,6 +13,14 @@ repositories {
     // https://jira.pentaho.com/browse/PDI-18983
 }
 
+configurations {
+    driver
+}
+dependencies {
+    driver 'mysql:mysql-connector-java:5.1.16'
```

Comment:
Not sure why JDBC driver would be required here... how about removing that here, and raising a separate issue or pull request about that?

Reply:
I will make a different PR for this - i am going to close this PR so as to make the requested change